locale/cs_CZ/foreman.po

```diff
@@ -10450,7 +10450,7 @@ msgid "You are using an unsupported browser."
 msgstr "Používáte nepodporovaný prohlížeč."
 
 msgid "You can add SSH public keys to a user in Foreman."
-msgstr "Ve Formean je možné přidat k uživateli veřejné části SSH klíčů."
+msgstr "Ve Foreman je možné přidat k uživateli veřejné části SSH klíčů."
```

Comment:
Not sure if we should be changing the translations directly, wouldn't it get overwritten on the next translation sync?

Reply:
yum, that's the same comment as for the js file, please drop this one from the PR and we're good to go :-)

Reply:
Done!